src/AudioIO.cpp

```diff
@@ -3211,7 +3214,7 @@ auto AudioIoCallback::AudioIOExtIterator::operator *() const -> AudioIOExt &
 
 void AudioIoCallback::StartAudioThread()
 {
-   mAudioThreadTrackBufferExchangeLoopRunning.store(true, std::memory_order_release);
+    mAudioThreadTrackBufferExchangeLoopRunning.store(true, std::memory_order_release);
```

Comment:
Indent changed.

I see StartAudioThreadAndWait was introduced at 00a8f2a68b1418b2f5979a7f33dc8092eb63b95a but never used.  And that contains the only other call to WaitForAudioThreadStarted.  That was an oversight in my earlier review.  Sorry.

The starting of the stream and the waiting for the ack are split, which isn't wrong.  So remove the unused function.

StopAudioThreadAndWait is unused too.

Reconsidering correctness of memory orderings, I think too that to be very correct, mAudioThreadTrackBufferExchangeLoopActive should be stored with release by AudioThread and tested with acquire by the main thread.  But maybe no difference as compiled for Intel.

Reply:
I just made a careful re-read of all of AudioIO::AudioThread.  Nothing more to say.